### PR TITLE
DDT-1064: Change CFT CBM -> FTQ MTQ

### PR DESCRIPTION
**Pull request by mp-assco**

### CODE REVIEW

File: datamodel/referencedata.d/unitofmeasures.csv [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-CFT,Cubic Feet
-CEL,Celsius
-FAH,Fahrenheit
-FOT,Foot
-MTR,Meter
+Unit of Measure Code,Unit of Measure Description
+KGM,Kilogram
+LBR,Pounds
+CBM,Cubic Metres
+CFT,Cubic Feet
```
Comment (HenrikHL) [inline datamodel/referencedata.d/unitofmeasures.csv:5]: These are the wrong values?!?

Comment (HenrikHL) [reply datamodel/referencedata.d/unitofmeasures.csv:5]: And I am not quite sure what has changed in this file - it appers everything has been updated...?

Comment (mp-assco) [reply datamodel/referencedata.d/unitofmeasures.csv]: Updated. Don't know why this was reverted.

**Comment by HenrikHL** [comment]
There are a lot of "units" missing checks - please add a check on them all (consignment_item, equiment, etc...)

**Comment by mp-assco** [comment]
> There are a lot of "units" missing checks - please add a check on them all (consignment_item, equiment, etc...)

Updated

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**